typescript/src/languages/en-US/commands/management.json

```
@@ -342,12 +344,12 @@
 			"here"
 		]
 	},
-	"setMemberLogsDescription": "Set the member logs channel.",
-	"setMemberLogsExtended": {
+	"setMemberAddLogsDescription": "Set the member add logs channel.",
+	"setMemberAddLogsExtended": {
 		"usages": [
 			"here/TextChannel"
 		],
-		"extendedHelp": "This command helps you setting up the member log channel. A member log channel only sends two kinds of logs: \"Member Join\" and \"Member Leave\".\nIf a muted user joins, it will send a special \"Muted Member Join\" event.\nAll messages are in embeds so you will need to enable the permission **{{EMBED_LINKS, permissions}}** for Skyra.\nYou also need to individually set the \"events\" you want to listen: `events.member-add` and `events.member-remove`.\nFor roles, you would enable `events.member-nickname-change` and/or `events.member-role-update` via the `config` command.",
+		"extendedHelp": "This command helps you setting up the member add log channel, which will send messages each time a user joins.\nIf a muted user joins, it will send a special \"Muted Member Join\" event.",
```

```suggestion
		"extendedHelp": "This command helps you setting up the member add log channel, which will receive a message each time a user joins.\nIf a muted user joins, it will send a special \"Muted Member Joined\" event.",
```

Didn't know channels were capable of sending messages

Also verify that you're sending that `\"Muted Member Joined\"`. The change is `join` -> `joined`. The member has already joined so it's past tense.